libs/ledgerjs/packages/hw-app-btc/src/Btc.ts

```diff
@@ -16,17 +16,22 @@ import { signP2SHTransaction } from "./signP2SHTransaction";
 import { checkIsBtcLegacy, getAppAndVersion } from "./getAppAndVersion";
 
 /**
- * Bitcoin API.
- *
+ * @class Btc
+ * @description Bitcoin API.
+ * @param transport The transport layer used for communication.
+ * @param scrambleKey This parameter is deprecated and no longer needed.
+ * @param currency The currency to use, defaults to "bitcoin".
  * @example
  * import Btc from "@ledgerhq/hw-app-btc";
  * const btc = new Btc({ transport, currency: "bitcoin" });
  */
 
 export default class Btc {
+  // Transport instance
   private _transport: Transport;
+  // The specific implementation used, determined by the nano app and its version.
+  // It chooses between BtcNew (new interface) and BtcOld (old interface).
   private _impl: BtcOld | BtcNew;
```

Comment:
Minor but could you provide info on where/in what context we use each instance ?

I saw below that btcOld is for versions earliers than 2.1.0 but where is that switch done ? :thinking: